doc-content/enterprise-only/getting-started-kogito/ref-kogito-embedded-process-decision-traffic-example.adoc

```diff
@@ -0,0 +1,138 @@
+----
+public class LicenseValidationService implements RuleUnitData {
+    private SingletonStore<Driver> driver;
+
+    public LicenseValidationService() {
+        this(DataSource.createSingleton());
+    }
+
+    public LicenseValidationService(SingletonStore<Driver> driver) {
+        this.driver = driver;
+    }
+
+    public void setDriver(SingletonStore<Driver> driver) {
+        this.driver = driver;
+    }
+
+    public SingletonStore<Driver> getDriver() {
+        return driver;
+    }
+
+    public Date getCurrentTime() {
+        return new Date();
+    }
+}
+----
+
+After validating the license, the traffic violation decision is created using DMN. The traffic violation decision verifies whether the driver is suspended or not. The traffic violation decision is declared in the `TrafficViolation.dmn` file.
+
+.Example `TrafficViolation.dmn` file
+image::KogitoMicroservices/traffic-violation-dmn.png[Image of Traffic Violation - DMN in DMN designer]
+--
+
+In the traffic violations project, the `Violation` class is mapped with the `speedLimit` attribute using the `@JsonProperty` annotation. Therefore, the `speedLimit` attribute can be integrated with the `Violation` data type defined in the DMN model.
+
+.`Violation` class defined in the traffic violations example
+[source,java]
+----
+public class Violation {
+
+    @JsonProperty("Code")
+    private String code;
+
+    @JsonProperty("Date")
+    private Date date;
+
+    @JsonProperty("Type")
+    private String type;
+
+    @JsonProperty("Speed Limit")
+    private BigDecimal speedLimit;
+
+    @JsonProperty("Actual Speed")
+    private BigDecimal actualSpeed;
+
+    //Getters / Setters
+}
+----
+
+.Violation data type in DMN
+image::KogitoMicroservices/violation-dmn-data-types.png[Image of Violation Data Type in DMN in DMN designer]
+
+In the traffic violations example, when the information of the driver is fetched, an external call to a service task is performed to retrieve the driver information from the database. Also, the service task implementation is performed using the `DriverService` class.
+
+In the data assignments of the service task, `driverId` is defined as an input variable and `driver` is defined as an output variable, consisting of the driver information. Fetching the driver information leads to the license validation task, which represents a task of calling a DRL service.
+
+The properties that are required to set the DRL service task include `Rule Language`, which must be set as `DRL` and the `Rule Flow Group`. You can set the value of `Rule Flow Group` using `unit:` + [the FQCN of the Rule Unit data class] format, such as `unit:org.kie.kogito.traffic.LicenseValidationService`. In the traffic violations example, the `driver` variable contains the license validation information.
+
+.License validation task data assignments
+image::KogitoMicroservices/license-validation-dmn-businessrule-data.png[Image of License Validation Task data assignment in process designer]
+
```

Comment:
Replace such as with for example, then consider the sentence that follows (For exmaple....

Reply:
Done